ui/redux/selectors/reactions.js

```diff
@@ -1,25 +1,17 @@
 import * as REACTION_TYPES from 'constants/reactions';
-import { selectClaimForUri } from 'redux/selectors/claims';
 
 const selectState = (state) => state.reactions || {};
 
 export const selectReactionsById = (state) => selectState(state).reactionsById;
+export const selectReactionsForClaimId = (state, claimId) => selectReactionsById(state)[claimId];
 export const selectFetchingReactions = (state) => selectState(state).fetchingReactions;
 
-export const selectReactionsForUri = (state, uri) => {
-  const claim = selectClaimForUri(state, uri);
-  const reactionsById = selectReactionsById(state);
-  return claim ? reactionsById[claim.claim_id] : {};
-};
-
-export const selectMyReactionForUri = (state, uri) => {
-  const claim = selectClaimForUri(state, uri);
-  const reactions = selectReactionsForUri(state, uri);
+export const selectMyReactionForClaimId = (state, claimId) => {
+  const reactions = selectReactionsForClaimId(state, claimId);
 
-  if (!claim || !reactions || !reactions.my_reactions) {
+  if (!reactions || reactions.my_reactions === null) {
     return undefined;
   }
```

Comment:
`=== null` negates the recent change that fixes the following (undefined `my_reaction`):

<img width="436" alt="image" src="https://user-images.githubusercontent.com/64950861/189814239-c7d709d2-64d9-4dad-aeda-41e948a2487f.png">

It is a merge error, or was the solution causing other problems?